packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -11,15 +11,48 @@ import { By } from "@angular/platform-browser";
 import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
 import { TranslateModule, TranslateService } from "@ngx-translate/core";
 import { NoopAnimationsModule } from "@angular/platform-browser/animations";
-import Spy = jasmine.Spy;
+import { OverlayContainer } from "@angular/cdk/overlay";
+
+@Component({
+	selector: `host-ng-control-component`,
+	template: `
+		<stark-dropdown
```

Comment:
The `disabled` input is not used here... however there is a test to check when `disable` changes: `Using reactive forms -> on change -> disabled` 😕